src/main/java/io/wisetime/connector/jira/database/JiraDb.java

```diff
@@ -36,31 +47,125 @@ public void asTransaction(final Runnable runnable) {
   }
 
   public boolean canUseDatabase() {
-    // TODO
-    return true;
+    return hasCorrectSchema(
+        ImmutableList.of(
+            Pair.of(
+                "jiraissue",
+                ImmutableSet.of("id", "issuenum", "summary", "timespent", "project")
+            ),
+            Pair.of(
+                "project",
+                ImmutableSet.of("id", "pkey")
+            ),
+            Pair.of(
+                "cwd_user",
+                ImmutableSet.of("user_name", "lower_email_address")
+            ),
+            Pair.of(
+                "worklog",
+                ImmutableSet.of("id", "issueid", "author", "timeworked", "created", "worklogbody")
+            ),
+            Pair.of(
+                "sequence_value_item",
+                ImmutableSet.of("seq_id", "seq_name")
+            )
+        )
+    );
+  }
+
+  private boolean hasCorrectSchema(List<Pair<String, Set<String>>> tableAndColumnPairs) {
```

Comment:
Two methods with the same name, one calling the other. We can omit this one and move its logic into `canUseDatabase`.